docs/src/main/asciidoc/security-basic-authentication-tutorial.adoc

```diff
@@ -72,20 +59,49 @@ You must also add your preferred database connector library.
 The instructions in this example tutorial use a PostgreSQL database for the identity store.
 ====
 
-* To add the Security Jakarta Persistence extension to an existing Maven project with Hibernate ORM, run the following command from your project base directory:
 
-:add-extension-extensions: security-jpa
-include::{includes}/devtools/extension-add.adoc[]
+=== Create the Maven project
 
-* To add the Security Jakarta Persistence extension to an existing Maven project with Hibernate Reactive, run the following command from your project base directory:
+You can either create a new Maven project with the Security Jakarta Persistence extension or you can add the extension to an existing Maven project. You can use either Hibernate ORM or Hibernate Reactive.
 
+* To create a new Maven project with the Jakarta Persistence extension, complete one of the following steps:
+** To create the Maven project with Hibernate ORM, use the following command:
++
+====
```

Comment:
I have no idea what this is in AsciiDoctor when you don't have a `[NOTE]`? I don't think we ever used that construct.
Does it bring something to the plate?

Reply:
@gsmet - Wrapping a numbered (. ) or bulleted (* ) step in a procedure (how-to/tutorial) by adding `====` to encapsulate code snippets fixes an alignment and numbering issues we are seeing both up and downstream.

For example, we were seeing '+' in the rendered output instead of a code snippet aligned and indented to match the current step, and the step numbering was broken or restarted back to *1.* again :
![image](https://github.com/quarkusio/quarkus/assets/92924207/3abab7fe-05f6-4607-a2cb-c3b97c414f4b)

By wrapping the code lines like so:
````
. The step 1 text goes here.
+
====
 code snippet include and attribute lines
====

. The next step goes here. If this has code, repeat and rinse:
+
====
code include etc
====

. The next step.... 
````
Glues all of the code lines for a step together and preserves the step numbering and indentation, for example:

![image](https://github.com/quarkusio/quarkus/assets/92924207/8765f680-34d1-4fb9-8129-2b7da5795859)

Does that make sense?

p.s my last commit introduced an asciidoc error. I am fixing now.... Thanks.

Reply:
Should add something to reference or howto doc (separate PR is fine) about how/when to use this construct to ensure numbered lists function properly.

Reply:
@michelle-purcell thanks for the explanation but I want to make sure you have tested this new layout on the website? We don't have everything AsciiDoctor implemented there so we need to make sure it looks good. Thanks for confirming.